src/app-layer-modbus.c

```diff
@@ -3262,6 +3269,95 @@ static int ModbusParserTest18(void) {
     FLOW_DESTROY(&f);
     PASS;
 }
+
+/** \test Send Modbus invalid function. */
+static int ModbusParserTest19(void) {
```

Comment:
can you convert this to use the FAIL_IF style macro's? New tests should have no branches at all.

Reply:
Done in new PR - Support Modbus Unit Identifier v5